bindings/python/mainloop.pyx

```diff
@@ -0,0 +1,13 @@
+
+cdef extern from "sol-mainloop.h":
+	int sol_init()
+	int sol_run()
+
+class SolettaMainloop():
+	def __init__(self):
+		sol_init()
+
```

Comment:
then provide a destructor that calls `sol_shutdown()`

It would be nice to also call `sol_args_set()` based on what you get from python, this way we can have some modules such as FBP's `app/argc-argv` to work.